### PR TITLE
Bugfix: ignore dynamics network when using the environment's dynamics

### LINKED ISSUE #66: Bug fix: ignore gradients for dynamics model in mapo_policy.compute_separate_gradients

### PR DESCRIPTION
**Pull request by 0xangelo**
Closes #66

### CODE REVIEW

File: mapo/agents/mapo/mapo_policy.py [hunk 1]
```diff
@@ -23,21 +23,24 @@ def build_mapo_losses(policy, batch_tensors):
...
-    policy.loss_stats["dynamics_loss"] = dynamics_loss
+    if dynamics_loss:
+        policy.loss_stats["dynamics_loss"] = dynamics_loss
     policy.loss_stats["critic_loss"] = critic_loss
     policy.loss_stats["actor_loss"] = actor_loss
     policy.mapo_losses = AgentComponents(
         dynamics=dynamics_loss, critic=critic_loss, actor=actor_loss
     )
-    return dynamics_loss + critic_loss + actor_loss
+    return (dynamics_loss if dynamics_loss else 0) + critic_loss + actor_loss
```
Comment (thiagopbueno) [inline mapo/agents/mapo/mapo_policy.py]: What about something along these lines?

```python
mapo_loss = critic_loss + actor_loss
if dynamics_loss is not None:
    mapo_loss += dynamics_loss
return mapo_loss
```

**Review by thiagopbueno — CHANGES_REQUESTED** [review]
